apps/web/src/styles/static-pages.scss

```
@@ -5,7 +5,12 @@
     display: block;
     width: 100% !important;
     max-width: 100% !important;
-    padding: 0 !important;
+    // Mobile reserves room for the fixed top bar; desktop keeps the full-bleed look.
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)